src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+    max-width: 250px;
+    word-wrap: break-word;
```

Comment:
The `word-wrap: break-word;` property is not needed here since you have already limited the width of `.card__title` with padding and max-width. This property is typically used to prevent overflow and ensure that long words do not break the layout, which should not be an issue with the current constraints.